core/src/main/scala/org/bykn/bosatsu/StringUtil.scala

```diff
@@ -42,9 +42,10 @@ abstract class GenericStringUtil {
     // We can ignore escaping the opposite character used for the string
     // x isn't escaped anyway and is kind of a hack here
     val ignoreEscape = if (quoteChar == '\'') '"' else if (quoteChar == '"') '\'' else 'x'
+    val tab = if (encodeTable.contains(quoteChar)) encodeTable else encodeTable.updated(quoteChar, s"\\$quoteChar")
```

Comment:
previously the ` character was not in the escape table, which means, internal versions were left unescaped. We need to ensure the quote character is in the replacement table.